### ISSUE TITLE
Divison by zero

### ISSUE THREAD

**Issue by maikwoehl** [Radivarig/UvSquares#29]
Hello,

I have just experienced an issue with the following line:

https://github.com/Radivarig/UvSquares/blob/9e7e65658a5609748cec2ec313cfa1f87451f704/uv_squares.py#L415

When two vertices are on the same coordinates due to extruding and accidentally not moving, there will be a division-by-zero error. 

Maybe displaying a report message about that possible mistake would be good.

``` python
if EXTEND_MODE == 'LENGTH_AVERAGE':
    try:
        fac = edge_lengths[l_b[2].edge.index][0] / edge_lengths[l_a[1].edge.index][0]
    except ZeroDivisonError:
        operator.report({'ERROR'}, "Two vertices have same coordinates! Aborting...")
        # Abort action...
elif EXTEND_MODE == 'LENGTH':
```
I hope that helps you.

Best regards and stay healthy,

Maik Wöhl

**Comment by Radivarig** [comment]
Hi Maik,


Thanks for reporting!

I believe I already opened an issue for showing the message, just forgot to
do it.

That definitely is a good idea, and also maybe to print out the command to
remove doubles/merge distance.

I'll look into it when I get home.

Best,
Reslav

**Comment by maikwoehl** [comment]
Issue # 25 is ment

**Comment by Radivarig** [comment]
No dedup needed, fixed in [commit#bb0090d0](https://github.com/Radivarig/UvSquares/commit/bb0090d0a5971a135ee7f2e9d1322dd003068392)
